boot/zephyr/Kconfig

```diff
@@ -71,6 +71,14 @@ config BOOT_UPGRADE_ONLY
 	  swapping them.  This prevents the fallback recovery, but
 	  uses a much simpler code path.
 
+config BOOT_BOOTSTRAP
+	bool "Boostrap erased slot0 from slot1"
+	default n
```

Comment:
I will skip all Zephyr/Kconfig suggestions, because this is the style already used which was initially added by Zephyr commiters, so if the idea is to change it or clean it up, someone should send a PR later cleaning up all options.